usaspending_api/submissions/serializers.py

```diff
@@ -7,4 +7,6 @@ class SubmissionAttributesSerializer(LimitableSerializer):
     class Meta:
 
         model = SubmissionAttributes
-        fields = '__all__'
+        fields = (
```

Comment:
@afrasier Thanks for raising this point. For transparency, will try to summarize what we discussed in the channel:

* we're agreed that we should move away from setting default serialization fields in the data models themselves
* as a team, we haven't yet discussed a new approach to verbosity/field limiting
* so in the meantime, this PR introduces some new serializers (program activity and object class) that are more basic than the rest

I feel okay about this awkward, in-between period for the serializers as long as:
* they don't apply to models that have their own endpoint (_e.g._, there's no `api/v1/object_class` yet)
* we're returning everything needed by the front-end
* we're not taking away a field that we previously were returning